angular/lib/src/lib/slider/slider.component.ts

```diff
@@ -0,0 +1,539 @@
+
+	/**
+	 * MouseDown event handler
+	 */
+	@Output('auMouseDown') mouseDown = new EventEmitter<MouseEvent>();
+
+	readonly _widget = callWidgetFactory(this.slider.api.registerHandle, null);
+	readonly widget = toSlotContextWidget(this._widget);
+	readonly api = this._widget.api;
+	readonly state: Signal<SliderHandleState> = toSignal(this._widget.state$, {requireSync: true});
+
+	constructor() {
+		this._widget.patch({
+			onMouseDown: (event: MouseEvent) => {
+				this._elementRef.nativeElement.focus();
+				this.slider._widget.patch({sliderDomRect: this._elementRef.nativeElement.parentElement.getBoundingClientRect()});
+				event.preventDefault();
+				const handleDrag = (e: MouseEvent) => {
+					e.preventDefault();
+					const newCoord = this.state().vertical ? e.clientY : e.clientX;
+					if (!this.readonly && !this.disabled) {
+						this._elementRef.nativeElement.focus();
+					}
+					if (this._prevCoordinate !== newCoord) {
+						this._prevCoordinate = newCoord;
+						this._zone.run(() => {
+							this.sliderActions.adjustCoordinate(newCoord, this.state().handleId);
+						});
+					}
+				};
+				this._zone.runOutsideAngular(() => {
+					document.addEventListener('mousemove', handleDrag);
+				});
+				// TODO mouse up doesn't work outside the handle area
+				document.addEventListener(
+					'mouseup',
+					() => {
+						document.removeEventListener('mousemove', handleDrag);
+					},
+					{once: true}
+				);
+
+				this.mouseDown.emit(event);
+			},
+			onKeyDown: (event: KeyboardEvent, handleId: number) => {
+				this.sliderActions.onKeydown(event, this.state().handleId);
+				this._zone.onStable.pipe(take(1)).subscribe(() => {
+					this._elementRef.nativeElement.focus();
+				});
+
+				this.keyDown.emit(event);
+			},
+		});
+	}
+}
+
+@Component({
+	selector: '[auSlider]',
+	standalone: true,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	encapsulation: ViewEncapsulation.None,
+	providers: [{provide: NG_VALUE_ACCESSOR, useExisting: forwardRef(() => SliderComponent), multi: true}],
+	imports: [SliderHandleComponent, NgIf, NgFor],
+	host: {
+		class: 'au-slider',
+		'[class]': 'state().vertical ? "au-slider-vertical" : "au-slider-horizontal"',
+		'(blur)': 'handleBlur()',
+	},
+	template: `
+		<div [class]="state().vertical ? 'au-clickable-slider-area-vertical' : 'au-clickable-slider-area'" (click)="sliderClick($event)"></div>
+		<div
+			*ngFor="let repeat of [].constructor(state().values.length - 1 || 1); let i = index"
+			#progress
+			class="au-slider-progress"
+			[attr.disabled]="state().disabled ? true : null"
+			[style.left.%]="state().progressLeft[i]"
+			[style.bottom.%]="state().progressBottom[i]"
+			[style.width.%]="state().progressWidth[i]"
+			[style.height.%]="state().progressHeight[i]"
+		></div>
+		<div
+			#minLabel
+			[class]="state().vertical ? 'au-slider-label-vertical au-slider-label-vertical-min' : 'au-slider-label au-slider-label-min'"
+			[style.visibility]="state().minValueLabelDisplay"
```

Comment:
By the way why is it hidden and not removed from DOM?

Reply:
the reziseObserver always needs it in the dom to return proper value for the DOMRect otherwise the computations fail

Reply:
Well then it makes sense to me have the `visible` / `hidden` and not boolean